blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -135,18 +122,20 @@ variable "repo_name" {
   default     = null
 }
 
-variable "sa_mlops_name" {
-  description = "Name for the MLOPs Service Account."
-  type        = string
-  default     = "sa-mlops"
-}
-
-variable "service_encryption_keys" { # service encription key
+variable "service_encryption_keys" {
   description = "Cloud KMS to use to encrypt different services. Key location should match service region."
   type = object({
-    bq      = string
-    compute = string
-    storage = string
+    aiplatform    = optional(string, null)
```

Comment:
`optional(string)`

Reply:
I have the default value to remove the try() in the code.

Reply:
`optional(string, null)` is the same as `optional(string)`

> The optional modifier takes one or two arguments.
>
>Type: (Required) The first argument specifies the type of the attribute.
> Default: (Optional) The second argument defines the default value that Terraform should use if the attribute is not present. This must be compatible with the attribute type. If not specified, **Terraform uses a null value of the appropriate type as the default**.